src/main/java/com/org/checkr/controller/UsersController.java

```diff
@@ -0,0 +1,26 @@
+package com.org.checkr.controller;
+
+import com.org.checkr.entity.User;
+import com.org.checkr.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/users")
+public class UsersController {
+
+    @Autowired
+    private UserService userService;
+
+    @GetMapping
+    public List<User> listAllUsers() {
+        return userService.getAllUsers();
```

Comment:
while returning users can use ResponseEntity for froviding hhtp status code for error handling